go/obsclient/obsclient.go

```diff
@@ -29,15 +26,3 @@ func NewObsClient(c rpc.Client) *ObsClient {
 func (oc *ObsClient) Close() {
 	oc.rpcClient.Stop()
 }
-
-// Blockchain Access
-
-// ChainID retrieves the current chain ID for transaction replay protection.
-func (oc *ObsClient) ChainID() (*big.Int, error) {
```

Comment:
Why get rid of this?

Reply:
Unused. Seemed arbitrary to just have that one?

Reply:
It's the client for unauthorised rpc requests., we just haven't added the rest yet.

Reply:
Ok, I'll readd.